receiver/gitproviderreceiver/internal/metadata/testdata/config.yaml

```diff
@@ -7,15 +7,13 @@ all_set:
       enabled: true
     git.repository.count:
       enabled: true
-    git.repository.pull_request.approved.time:
+    git.repository.pull_request.count:
       enabled: true
-    git.repository.pull_request.merged.count:
+    git.repository.pull_request.open_time:
```

Comment:
I'm thinking that `open_time` could more closely match the syntax of `time_to_*` by renaming it to `time_open`. 

However, I'm also thinking that in general, these metrics will be subject to changing in order to better matching the conventions from the CI/CD Working Group. I'm not what level of change will actually be yet, but just something to keep in the back of our mind. Thoughts @astencel-sumo?

Reply:
I'm fine with either way, especially that the component is in development stability status.

Reply:
I'd like to not have new sem conv's block the move of this component from development to alpha tbh. I think the contribution of this upstream has taken a bit longer than I would've liked as is (it's a me problem I promise). So would really like to knock out the github portion, get to alpha built into distro, then add in the gitlab side & in the future as metric conventions come out, we'll be able to adjust metric names better to match those conventions (which this is definitely informing). 

On the original point for renaming `open_time` to `time_open` I think I like the consistency and readability of `time_open` better.